updated 8th sept blog for office hours (#62)
kubeslice/docs: blog/2022-09-8-office-hours.mdx

---

> The [KubeSlice project](https://github.com/kubeslice) recently hosted the third episode of its Office Hours series on Thursday, 8th September, 2022

Reviewer: ```suggestion
The [KubeSlice project](https://github.com/kubeslice) recently hosted the third episode of its Office Hours series on Thursday, 8th September 2022
```

Reviewer: no need to provide this comma

> In this episode, [Eric Peterson](https://www.linkedin.com/in/eric-peterson-90073b2/), Vice President Of Engineering at [Avesha](https://avesha.io/) walked us through the road map, introduced us to multi-cluster, which is one of the basic issues, KubeSlice is attempting to address and demonstrated the connectivity to Kubernetes as well as Kubernetes' support for multiple tenants works.

Reviewer 2: ```suggestion
In this episode, [Eric Peterson](https://www.linkedin.com/in/eric-peterson-90073b2/), Vice President Of Engineering at [Avesha](https://avesha.io/) walked us through the road map and introduced us to the challenges of multi-cluster multi-tenancy in Kubernetes that KubeSlice is trying to address.
```

> You can jump in here, the [KubeSlice Roadmap](https://github.com/orgs/kubeslice/projects/4/views/1), and feel free to take up any issue and again you can reach out on the  #kubeslice channel on the [Kubernetes slack](https://slack.k8s.io/) where we hang out, if you have questions about something or if you need any guidance and support. 

Reviewer 2: ```suggestion
He walked us through the various issues on [KubeSlice roadmap](https://github.com/orgs/kubeslice/projects/4/views/1) and explained the context behind the creation of each one. If there's anything in particular that you feel you could work on, feel free to assign it on your name. If you are unable to, please reach out to us on the  **#kubeslice** channel in the [Kubernetes slack](https://slack.k8s.io/). 
```

Additionally, the Office Hours are intended 
to be as interactive as possible and we’d love you to join us on our next one to be held on 22nd September 2022. 

> To receive an invite for the next one in your inbox & stay updated with the latest goings-on in the project, [join our Google Group](https://groups.google.com/g/kubeslice-dev).

Reviewer: invite -> invitation , looks better here

Reviewer 2: I disagree because you do not send out meeting invitations, you send out meeting invites.
